plugins/colorbutton/plugin.js

```diff
@@ -402,7 +402,7 @@
 
 				output.push( '</tr></tbody></table>' );
 
-				return output.join( '' );
+				return CKEDITOR.tools.htmlSafeByReview(output.join( '' ), 'renderColors is a function that outputs html created internally using values from editor or lang config.');
```

Comment:
Remind me, did we make a decision on whether these `output` values from the configs were `htmlSafeByReview` or some sort of legacy conversion? (I believe you that you traced to make sure `output` was from configs, so just want to make sure we're being consistent with everywhere else that does `output.join('')`.)

Reply:
SGTM